Makefile

```diff
@@ -11,3 +11,7 @@ test-verbose:
 testing:
 	$(TESTER) $(OPTS) --watch $(TESTS)
 .PHONY: test docs
+
+b benchmarks:
```

Comment:
This is not a typo, Makefiles allow multi-target, so instead of typing `make benchmarks` all the time, I like to alias `b` so you can type `make b` or `make benchmarks`. Seeing this now, I should also add `benchmark` in case the user forgets to type the `s` in `benchmarks`